### ISSUE TITLE
Strengthen motivation for procurement language

### ISSUE THREAD

**Issue by judbrew** [w3c/maturity-model#323]
The current text seems to under-motivate the strategic importance of procurement policy in an organization's accessibility ecosystem, and as feedback towards improving the offerings of product vendors. Please consider clarifying "When accessibility criteria..." sentence and expanding the motivations along the following lines. 

    .... When accessibility criteria are integrated as an essential consideration of procurement processes and contract language, an organization can be more capable of providing accessible products, services, and workplaces. It can save time through more efficient production of accessible content, and avoid unnecessary time spent in remediation of inaccessible content created by tools that do not support accessibility. Additionally, if an organization communicates its product selection criteria to product vendors, it can help incentivize product developers to improve their future product offerings.

**Comment by jeffkline** [comment]
Thank you for your comments. Please see this proposed rewrite of Section 3.6 Procurement incorporating your input.

Procurement is a strategic process focused on finding and acquiring cost-effective products needed by an [organization](https://w3c.github.io/maturity-model/#dfn-organization). Activities in procurement include sourcing, negotiation, and selection of goods and services.

The majority of an organization's ICT assets result from procurement transactions and contracts. Therefore, accessibility criteria must be integrated as an essential consideration of procurement processes, contract language, etc. This ensures that an organization will be more capable of providing accessible products, services, and workplaces while reducing time, cost and risks of deploying inaccessible IT deliverables.  Additionally, when an organization communicates to product vendors that accessibility requirements are part of the product selection criteria, it can help incentivize product developers to improve their future product offerings.

**Comment by Helixopp** [comment]
@judbrew here is what we came up with to address this issue. Please, let us know if this satisfies your request:

Adhering to accessibility principles in procurement saves time by enabling more efficient production of accessible content and by avoiding unnecessary remediation of inaccessible content created by tools that do not support accessibility. It also reduces the need for accommodations and increases employment among people with disabilities, thereby
08:18 <Sheri_B-H> making accessibility more ingrained within an organization. Additionally, when an organization communicates to product vendors that accessibility requirements are part of the product selection criteria, it can incentivize other organizations to improve their future offerings.

**Comment by judbrew** [comment]
David, thank you for checking. This response only minimally addresses the point that I was raising; it addresses it only within the context of a single organization, or other individual organizations that might be influenced by a single organization that uses its procurement policies strategically. It does not capture the strength of coordinated procurement policies that selectively purchase accessibility-supporting software and apps instead of apps that continue to perpetuate accessibility barriers in the process of creating websites, apps, and content. What is your timeline for resolving comments on the latest version of the maturity model? It would be helpful if I could look at the updated draft, to ensure that I am proposing language harmonized with your current document—but I would appreciate the opportunity to propose a modified version of this language for your consideration, to try to better capture the point that I feel is still missing in this. (Difficult today, but possible over the coming weekend.)On Jun 17, 2026, at 11:21 AM, David Fazio ***@***.***> wrote:﻿Helixopp left a comment (w3c/maturity-model#323)
@judbrew here is what we came up with to address this issue. Please, let us know if this satisfies your request:
Adhering to accessibility principles in procurement saves time by enabling more efficient production of accessible content and by avoiding unnecessary remediation of inaccessible content created by tools that do not support accessibility. It also reduces the need for accommodations and increases employment among people with disabilities, thereby
08:18 <Sheri_B-H> making accessibility more ingrained within an organization. Additionally, when an organization communicates to product vendors that accessibility requirements are part of the product selection criteria, it can incentivize other organizations to improve their future offerings.

**Comment by Helixopp** [comment]
Thank you, Judy. We have made decisions regarding other sections of the document, as we work through Github Issues, that are not yet reflected in the Editors Draft. However, we do not believe that any of those changes will fundamentally alter anything related to Procurement.

We would appreciate your proposed language anytime. Please, take into consideration that this non-normative document focuses on "the what" not the "why" or the "how".
